Basic/ClientDriven/Assets/Scripts/NetworkObjectSpawner.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.Networking;
+using Random = System.Random;
+
+public class NetworkObjectSpawner : NetworkBehaviour
+{
+    public NetworkObject prefabReference;
```

Comment:
please use `[SerializeField]` instead of `public` if the purpose is only to expsoe ths field in the inspector. `public` should be used when needed (= when you want something to be accessible from other namespaces).